WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -633,16 +633,20 @@ public static void viewActivityLogDetailForResult(
     ) {
         Map<String, Object> properties = new HashMap<>();
         properties.put(ACTIVITY_LOG_ACTIVITY_ID_KEY, activityId);
+        String source = null;
```

Comment:
1) Minor (🔍): Consider removing the optional ` = null`.
2) Minor (🔍): Consider reusing the `TRACK_EVENT_PROPERTY_VALUE` instead of duplicating them. This will make the code more robust and less error-prone in the future. The below example removes an extra duplication as well:

```
String source;
if (rewindableOnly) {
    source = BACKUP_TRACK_EVENT_PROPERTY_VALUE;
} else {
    source = ACTIVITY_LOG_TRACK_EVENT_PROPERTY_VALUE;
}
properties.put(SOURCE_TRACK_EVENT_PROPERTY_KEY, source);
```